shs/update_td_b_up.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+cd $HOME
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)